data/os/RedHat.yaml

```diff
@@ -1,3 +1,9 @@
 ---
+profile_monitoring::telegraf::config_dirs:
```

Comment:
Because I only know where these config directories are for RHEL. If we extend to other OSes, I wasn't wanting to presume we knew where they would be installing these config files and what user/group is runs as based on the telegraf package.

But I could be talk into moving it into the common.yaml file.

Reply:
I think it's safe to throw those into common.yaml. I installed telegraf on my test Ubuntu VM and looks like the file layout is the same.

Reply:
All of the `profile_monitoring::telegraf::config_dirs*` parameters are now moved into `common.yaml`.

Reply:
Looks good, thanks!